frame/call-filter/src/validation.rs

```diff
@@ -0,0 +1,23 @@
+use composable_support::validation::Validate;
+use support::{pallet_prelude::*};
+use crate::pallet::{Config, CallFilterEntryOf};
+
+#[derive(Debug, Decode, Copy, Clone)]
+pub struct ValidEntry<T>{
+    _marker: PhantomData<T>,
+}
+
+impl<T: Config> Validate<CallFilterEntryOf<T>, ValidEntry<T>> for ValidEntry<T> {
+
+    fn validate(input: CallFilterEntryOf<T>) -> Result<CallFilterEntryOf<T>, &'static str>{
```

Comment:
no sure why separate rs file is needed for this? may be retain in lib.rs or move into types.rs this and CallFilterEntryOf